### PR TITLE
Feature #66 -  Create "review" model

### PR DESCRIPTION
**Pull request by OleksiiDatsiuk**
Created "review" model

### CODE REVIEW

File: beauty/api/models.py [hunk 1]
```diff
@@ -105,3 +115,30 @@ def __repr__(self):
...
+
+class Review(models.Model):
+    """This class represents Review entity"""
+    text_body = models.CharField(
+        max_length=500,
+        verbose_name="Review text body"
+    )
+    rating = models.IntegerField(
+        blank=False,
+        validators=(MinValueValidator(0), MaxValueValidator(5)),
```
Comment (datsiuk-ira) [inline beauty/api/models.py:131]: as I know, min and max values must be in [ ], not in ( ), to include all numbers from min to max, not only them. check this, may it works okay with ( ) too.

Comment (OleksiiDatsiuk) [reply beauty/api/models.py:131]: For "validators" you can use both tuple and list. I prefer using tuple, cuz it is way faster than list

File: beauty/api/models.py [hunk 2]
```diff
@@ -105,3 +115,30 @@ def __repr__(self):
         :return: class, id
         """
         return f'{self.__class__.__name__}(id={self.id})'
+
+
+class Review(models.Model):
+    """This class represents Review entity"""
```
Comment (pazuzu-ua) [inline beauty/api/models.py]: Need to implement a docstring

File: beauty/api/models.py [hunk 3]
```diff
@@ -105,3 +115,30 @@ def __repr__(self):
...
+        verbose_name="Review text body"
+    )
+    rating = models.IntegerField(
+        blank=False,
+        validators=(MinValueValidator(0), MaxValueValidator(5)),
+        verbose_name="Review rating"
+    )
+    date_of_publication = models.DateTimeField(
+        auto_now_add=True,
+        verbose_name="Review time of publication"
```
Comment (pazuzu-ua) [inline beauty/api/models.py]: "Time of review publication"

Comment (pazuzu-ua) [inline beauty/api/models.py]: "Time of review publication"

File: beauty/api/models.py [hunk 4]
```diff
@@ -105,3 +115,30 @@ def __repr__(self):
         :return: class, id
         """
         return f'{self.__class__.__name__}(id={self.id})'
+
+
+class Review(models.Model):
+    """This class represents Review entity"""
+    text_body = models.CharField(
+        max_length=500,
+        verbose_name="Review text body"
```
Comment (pazuzu-ua) [inline beauty/api/models.py]: "Review text"

Comment (pazuzu-ua) [inline beauty/api/models.py]: "Review text"

File: beauty/api/models.py [hunk 5]
```diff
@@ -105,3 +115,30 @@ def __repr__(self):
...
+    )
+    date_of_publication = models.DateTimeField(
+        auto_now_add=True,
+        verbose_name="Review time of publication"
+    )
+
+    def __str__(self):
+        return self.text_body
+
+    class Meta:
```
Comment (pazuzu-ua) [inline beauty/api/models.py:155]: Maybe add ordering?

Comment (pazuzu-ua) [inline beauty/api/models.py:155]: Maybe add ordering?

**Pull request merged**